app/styles/partials/rails.scss

```diff
@@ -0,0 +1,57 @@
+@media screen and (min-width : 1200px) {
+    .ui .rail.left {
+      left : auto;
+      right : 80%;
+      padding : 0 2rem 0 0;
+      margin : 0 2rem 0 0;
+    }
+    .event .description {
```

Comment:
Like I said before, please ensure this works on all pages with a rail. Use css [Adjacent sibling selectors](https://developer.mozilla.org/en/docs/Web/CSS/Adjacent_sibling_selectors) to make this work with all pages that have a rail.